openstack/baremetalintrospection/noauth/doc.go

```diff
@@ -0,0 +1,16 @@
+package noauth
```

Comment:
We're using the Go packaging format when the `package something` directive goes after the `/* Package something...` comment.
See [containerinfra/v1/clusters/doc.go](https://github.com/gophercloud/gophercloud/blob/master/openstack/containerinfra/v1/clusters/doc.go) for the example.
Also there is an example in [godoc-documenting-go-code](https://blog.golang.org/godoc-documenting-go-code).

Reply:
Oh, I see that it will be bad for consistency with the other baremetal stuff already merged into Gophercloud, so ignore my previous comment 😄

Reply:
Ah, interesting - I never noticed this before. I can get this cleaned up soon.